src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
```

Comment:
```suggestion
```

Your tests failed because of this. Don't set top margin. Better set padding-bottom: 50px for its parent element (main with class `main`)